charts/app-config/values-integration.yaml

```diff
@@ -1901,6 +1901,9 @@ govukApplications:
         - name: reschedule-pubs-after-db-restore  # non-prod only
           task: "editions:requeue_scheduled_for_publishing"
           schedule: "38 7 * * 1-5"
+        - name: publish-missed-scheduled-editions
+          task: "editions:publish_missed_scheduled_editions"
+          schedule: "15 5 * * * *"
```

Comment:
I think there's an extra asterisk in each of these schedules. I believe there should be 5 components, where these each have 6.